README.md

```diff
@@ -128,3 +128,18 @@ Tests are written with [mocha](https://mochajs.org)
 ```bash
 npm test
 ```
+
+# Integration with TypeScript
+
+DRange includes TypeScript definitions.
+
+```typescript
+import * as RandExp from "randexp";
+const randExp = new RandExp(/[a-z]{6}/);
```

Comment:
Would prefer all lowercased variable to have it consistent with the rest of the examples on the readme